Dockerfile

```diff
@@ -1,61 +1,59 @@
-FROM phusion/baseimage:0.9.15
+FROM phusion/baseimage:0.9.18
 MAINTAINER Open Knowledge
 
-# Disable SSH
-RUN rm -rf /etc/service/sshd /etc/my_init.d/00_regen_ssh_host_keys.sh
-
-ENV HOME /root
+# Environment
 ENV CKAN_HOME /usr/lib/ckan/default
 ENV CKAN_CONFIG /etc/ckan/default
 ENV CKAN_DATA /var/lib/ckan
+ENV CKAN_GIT_BRANCH release-v2.5.2
```

Comment:
I'd rather not build the 2.5 image from the master branch, and I don't like downloading the source as a tarball from the Dockerfile.

Docker hub automated builds should build the branch that triggered them. CKAN 2.6 will be the first release with this docker configuration, so we can start docker hub automated builds for stable releases after 2.6 is out by just adding that branch to docker hub's automated builds. If you'd like to build 2.5 (and earlier) then we can consider back-porting just the docker configuration once you're happy with it.

I don't think it would ever make sense for a stable docker image to change just because the docker configuration on master changed. If there is a fix necessary we can backport it along with our other bug fixes.